apertium-tur-uzb.tur-uzb.dix

```diff
@@ -4434,13 +4476,13 @@
        <e><p><l>nişan<s n="n"/></l><r>orden<s n="n"/></r></p></e>
        <e><p><l>karın<s n="n"/></l><r>qorin<s n="n"/></r></p></e>
        <e><p><l>üstün<s n="adj"/></l><r>ustun<s n="adj"/></r></p></e>
-       <e><p><l>sütün<s n="n"/></l><r>sutun<s n="n"/></r></p></e>
+       <e><p><l>sütün<s n="n"/></l><r>ustun<s n="n"/></r></p></e>
        <e><p><l>salyangoz<s n="n"/></l><r>shilliq<s n="n"/></r></p></e>
        <e><p><l>olma<s n="n"/></l><r>boʻlganliq<s n="n"/></r></p></e>
```

Comment:
olma?? This not exist in Turkish as a noun, what the meaning of boʻlganliq?

Reply:
Well, probably it's a mistake. "boʻlganliq" is incorrect too. Removed it.